Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -920,7 +920,6 @@ def sanitize_ARGV_and_ENV
     end
 
     ENV["HOMEBREW_DEVELOPER"] = "1"
-    ENV["HOMEBREW_NO_AUTO_UPDATE"] = "1"
```

Comment:
"These times are now fast enough to avoid any further special-casing for `--preinstall`"  in commit message